### PR TITLE
LOGDS-22 Add script for removal of unassigned shifts

### PR DESCRIPTION
**Pull request by ssuerbier**

### CODE REVIEW

File: staffing/util/util.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import datetime
+
+import pendulum
+
+MODE_TO_DURATION = {"DAY": pendulum.duration(days=1), "WEEK": pendulum.duration(weeks=1)}
+
+
+def get_timeframe(date: pendulum.Date, mode: str, timezone: pendulum._Timezone):
```
Comment (ThiagoVieira) [inline staffing/util/util.py]: return Types?

File: staffing/api/rooster_api.py [hunk 2]
```diff
@@ -143,7 +143,10 @@ def create_open_shifts(self, shifts: typing.List[OpenShift], published=False) ->
 
         return [f.result() for f in futures]
 
-    def get_open_shifts(self, starts_at: datetime.datetime, ends_at: datetime.datetime, city_id: int=None, starting_point_id: int=None, exclude_non_demand: bool = None):
+    def get_open_shifts(self, starts_at: pendulum.DateTime,
+                        ends_at: pendulum.DateTime, city_id: int=None,
+                        starting_point_id: int=None,
+                        exclude_non_demand: bool=True):
         filters = {
```
Comment (ThiagoVieira) [inline staffing/api/rooster_api.py]: On the function def, maybe use Optional[int] for the city_id and starting point Id.  In the function def of api_client.list, the filters var could be typed to expect an Optional list of type any as the value param for the keys in the dictionary.

File: staffing/util/util.py (hunk 1 above)
Comment (ThiagoVieira) [inline staffing/util/util.py]: return type

File: staffing/api/rooster_api.py (hunk 2 above)
Comment (ssuerbier) [reply staffing/api/rooster_api.py]: I guess you mean `typing.Dict[str, typing.Any]`.

**Pull request merged**